joint_trajectory_controller/src/cartesian_trajectory_generator.cpp

```diff
@@ -398,12 +399,28 @@ bool CartesianTrajectoryGenerator::read_state_from_hardware(JointTrajectoryPoint
   state.positions[4] = orientation_angles[1];
   state.positions[5] = orientation_angles[2];
 
-  state.velocities[0] = measured_state->twist.twist.linear.x;
-  state.velocities[1] = measured_state->twist.twist.linear.y;
-  state.velocities[2] = measured_state->twist.twist.linear.z;
-  state.velocities[3] = measured_state->twist.twist.angular.x;
-  state.velocities[4] = measured_state->twist.twist.angular.y;
-  state.velocities[5] = measured_state->twist.twist.angular.z;
+  ////// Convert twist in body frame to world frame since CTG/JTC expects state in world frame
```

Comment:
Would it make sense to say something like what was explained in https://github.com/SchillingRobotics/ros2_controllers-stogl/pull/9#issuecomment-1707227740 here in case anyone without CTG/JTC background (like me :upside_down_face:) looks at this code later?